docs/dev/tools/cli.md

```diff
@@ -970,7 +982,6 @@ See [Integrate other hardware](/operate/get-started/other-hardware/) and [Update
 You can use the `module build start` or `module build local` commands to build your custom module according to the build steps in your <file>meta.json</file> file:
 
 - Use `build start` to build or compile your module on a cloud build host that might offer more platform support than you have access to locally.
-  Your repository must be public to use the `build start` command.
```

Comment:
Why are you removing this?

Reply:
Whoa--this was not in the diff here https://app.gopromptless.ai/change-history/a33eb356-c872-4b9b-805f-3f5c67cd6232